src/store/actions/account.actions.ts

```diff
@@ -0,0 +1,55 @@
+import { createAction, props } from '@ngrx/store';
+
+
+export enum AuthActionTypes {
+  LOGIN = '[Auth] Login',
+  LOGIN_SUCCESS = '[Auth] Login Success',
+  LOGIN_FAILURE = '[Auth] Login Failure',
+  REGISTER = '[Auth] Register',
+  REGISTER_SUCCESS = '[Auth] Register Success',
+  REGISTER_FAILURE = '[Auth] Register Failure',
+  LOGOUT = '[Auth] Logout',
+}
+
+export const LogIn = createAction(
```

Comment:
pls, use camelCase instead of PascalCase for actions with such syntax

Reply:
Fixed in A3